packages/flutter/lib/src/cupertino/picker.dart

```diff
@@ -162,6 +170,10 @@ class CupertinoPicker extends StatefulWidget {
   /// A [FixedExtentScrollController] to read and control the current item, and
   /// to set the initial item.
   ///
+  /// By default, a [CupertinoPicker] will scroll to tapped items. However, if
+  /// this parameter is provided, it is expected to handle the picker's own
+  /// tap-to-scroll behavior.
```

Comment:
If we cannot find a way to support the user provided one, can we add a sample that shows folks how to do it?